_shared/project/.github/workflows/ci.yml

```diff
@@ -56,6 +56,8 @@ jobs:
         image: postgres:{{ cookiecutter.get("__postgres_version", "15.3-alpine") }}
         ports:
         - {{ cookiecutter['__postgres_port'] }}:5432
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
This should have been in the earlier PR that upgraded the cookiecutter's default Postgres version (https://github.com/hypothesis/cookiecutters/pull/133): this envvar is needed in `ci.yml` otherwise Postgres won't start up in CI.